packages/gatsby/src/state-machines/query-running/actions.ts

```diff
@@ -1,6 +1,7 @@
 import { IQueryRunningContext } from "./types"
 import { DoneInvokeEvent, assign, ActionFunctionMap } from "xstate"
 import { enqueueFlush } from "../../utils/page-data"
+import { callApi, markNodesDirty } from "../develop/actions"
```

Comment:
Importing from develop actions doesn't feel like the correct thing to do. So I am open to suggestions here!